sdk/forms/src/main/assets/klaviyo-forms-helpers.js

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
Since I was able to put this in an actual JS file, it'd be awesome to even have some JS tests in our CI. Maybe not a realistic or wholly necessary thing, given how simple these methods are. But the cool factor...